src/signin.css

```
@@ -110,6 +110,11 @@ input:focus {
   font-size: 1.4rem;
 }
 
+.social-logo {
+  display: flex;
+  gap: 1.6rem;
+}
```

```suggestion
.social-logo-container {
  display: flex;
  gap: 1.6rem;
}
```
`display: flex`를 사용하니까 flexbox이고, 마크업에서도, 구글/카카오 로고 2가지를 담은 컨테이너 역할을 하고 있기 때문에, 위와 같이 제안드려요